src/translations.php

```diff
@@ -393,6 +393,15 @@
         "Present" => "Presente",
         "Excluding {days}" => "Exceto {days}",
     ],
+    "ro_cuDiacritice" => [
```

Comment:
Any reason to call this `ro_cuDiacritice` instead of `ro` for simplicity? Would there need to be other versions of Romanian supported separately?

```suggestion
    "ro" => [
```

Reply:
Yes, currently the script expects the locale to have a language code (2 to 3 letters), optionally followed by a script code (4 letters), optionally followed by a region code (2 letters or 3 numbers), for example, "ro", "ro_RO", "ro_MD" are options, sometimes we use scripts like "zh_Hans" to differentiate from "zh_Hant", but in general we only specify region and script if it's not the "typical" for this language. We can change the normalization if we need to, but in this case, I _think_ Romanian should just be "ro" because (as far as I know) cu diacritice is the only version needed for Romanian.

https://github.com/DenverCoder1/github-readme-streak-stats/blob/main/src%2Fcard.php#L265